RecoTauTag/Configuration/python/HPSPFTaus_cff.py

```diff
@@ -584,7 +584,7 @@
 hpsPFTauDiscriminationByIsolationMVArun2v1PWoldDMwLTraw = hpsPFTauDiscriminationByIsolationMVArun2v1DBoldDMwLTraw.clone(
     mvaName = cms.string("RecoTauTag_tauIdMVAPWoldDMwLTv1"),
     mvaOpt = cms.string("PWoldDMwLT"),
-    srcPUcorrPtSum = cms.InputTag('hpsPFTauNeutralIsoPtSumWeight'),
+    srcNeutralIsoPtSum = cms.InputTag('hpsPFTauNeutralIsoPtSumWeight'),
```

Comment:
is this a name change wrt to master? Is the variable the same thing or did it also change content?

Reply:
This is a fix that I picked up from @mbluj, who should know better where it comes from and whether it should enter central cms-sw in a different way.